x/ccv/provider/keeper/gov_hook.go

```diff
@@ -0,0 +1,93 @@
+package keeper
```

Comment:
For consistency, I suggest either renaming `hooks.go` to `staking_hooks.go` or merging this file to it.